### PR TITLE
Remove curl from the prerequisites

### PR DESCRIPTION
**Pull request by toliaqat**
Considering wget is by default installed on most OSes. Let's use it and remove curl from the `Prerequisites`.

### CODE REVIEW

File: main/guides/getting-started/README.md [hunk 1]
```diff
@@ -18,35 +18,14 @@ Currently Agoric supports [Windows Subsystem for Linux](https://learn.microsoft.
...
-```
-
-</details>
-
-
 ### Installing NVM and Node
 <details>
 <summary>Installing NVM and Node on Ubuntu</summary>
 
 At this point the Node Version Manager (NVM) utility will be installed. `nvm` makes it easy to select the specific version of [Node](https://nodejs.org/) that will be required for this tutorial (v18.16.0).
```
Comment (gibson042) [inline main/guides/getting-started/README.md:25]:
```suggestion
At this point the [Node Version Manager](https://github.com/nvm-sh/nvm) (NVM) utility will be installed. `nvm` makes it easy to select the specific version of [Node](https://nodejs.org/) that will be required for this tutorial (v18.16.0).
```

File: main/guides/getting-started/README.md [hunk 2]
```diff
@@ -18,35 +18,14 @@ Currently Agoric supports [Windows Subsystem for Linux](https://learn.microsoft.
...
 
 ```sh
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
+wget -O - https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
```
Comment (gibson042) [inline main/guides/getting-started/README.md:28]:
`````suggestion
curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
```
or
```sh
wget -O- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.7/install.sh | bash
`````

**Review by gibson042 — COMMENTED** [review]
I think `curl` is more standard for this kind of thing (e.g., note its use below for installing Docker), but really we should just defer to the [nvm instructions](https://github.com/nvm-sh/nvm?tab=readme-ov-file#installing-and-updating) (which cover both utilities).

File: main/guides/getting-started/README.md (hunk 1 above)
Comment (kbennett2000) [reply main/guides/getting-started/README.md]: Makes sense to me, let's go with wget!

**Review by kbennett2000 — APPROVED** [review]

**Pull request merged**